### PR TITLE
Update core-committers.md

### PR DESCRIPTION
**Pull request by comharris**

### CODE REVIEW

**Comment by comharris** [comment]
@gsagula
@ethervoid

Not urgent, feel free to update your dev areas

**Comment by jasonblais** [comment]
Some of these may also need to be added to https://developers.mattermost.com/internal/rd-teams/

File: site/content/contribute/getting-started/core-committers.md [hunk 1]
```diff
@@ -122,6 +122,31 @@ Below is the list of core committers working on Mattermost:
...
+    - Dev areas: TBA
+- **<a name="agniva.de.sarker">Agniva De Sarker</a>**
+    - @agniva.de.sarker on [community.mattermost.com](https://community.mattermost.com/core/messages/@agniva.de.sarker) and [@agnivade](https://github.com/agnivade) on GitHub
+    - Dev areas: Performance, Load Tests, Go internals
+- **<a name="chris.overton">Chris Overton</a>**
+    - @chris.overton on [community.mattermost.com](https://community.mattermost.com/core/messages/@chris.overton) and [@chris-overton](https://github.com/chris-overton) on GitHub
+    - Dev areas: Cloud, Elasticsearch, REST API, Kubernetes
+ - **<a name="gabriel.sagula">Gabriel Sagula</a>**
+    - @gabriel.sagula on [community.mattermost.com](https://community.mattermost.com/core/messages/@gabriel.sagula) and [@gsagula](https://github.com/gsagula) on GitHub
+    - Dev areas: TBA
```
Comment (jwilander) [inline site/content/contribute/getting-started/core-committers.md]:
```suggestion
    - Dev areas: Kubernetes, Cloud
```

**Comment by gsagula** [comment]
Sorry, I was travelling. +1 on @jwilander suggested changes, otherwise it LTGM.

File: site/content/contribute/getting-started/core-committers.md [hunk 2]
```diff
@@ -122,6 +122,31 @@ Below is the list of core committers working on Mattermost:
...
+    - Dev areas: Mobile
+- **<a name="ian.whitlock">Ian Whitlock</a>**
+     - @ian.whitlock on [community.mattermost.com](https://community.mattermost.com/core/messages/@ian.whitlock) and [@gigawhitlocks](https://github.com/gigawhitlocks) on GitHub
+    - Dev areas: Cloud
+- **<a name="juho.nurminen">Juho Nurminen</a>**
+    - @juho.nurminen on [community.mattermost.com](https://community.mattermost.com/core/messages/@juho.nurminen) and [@jupenur](https://github.com/jupenur) on GitHub
+    - Dev areas: Security
+- **<a name="mario.de.frutos">Mario de Frutos</a>**
+    - @mario.de.frutos on [community.mattermost.com](https://community.mattermost.com/core/messages/@mario.de.frutos) and [@ethervoid](https://github.com/ethervoid) on GitHub
+    - Dev areas: TBA
```
Comment (ethervoid) [inline site/content/contribute/getting-started/core-committers.md]:
```suggestion
    - Dev areas: Server, Load Tests
```

**Comment by jasonblais** [comment]
@grundleborg I'll go ahead and merge as Camille is looking to submit another PR for new hires tomorrow. You can review / suggest updates if any post-merge. 

Thanks @comharris for the PR!

**Pull request merged**